packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -112,6 +112,8 @@ private static BitmapDescriptor toBitmapDescriptor(
         }
         final Map<?, ?> byteData = toMap(data.get(1));
         return getBitmapFromBytes(byteData, density, new BitmapDescriptorFactoryWrapper());
+      case "null":
```

Comment:
Basically, because the tests need to be able to construct Markers, which means every non-nullable field must have a value assigned, and it therefore must be able to assign a non-null `BitmapDescriptor`. I initially tried using `defaultMarker`, but that apparently does not work until the `BitmapDescriptorFactory` is initialized in some way, which is not done in the tests. Do you have any preferable alternatives in mind?

Reply:
Can you make `BitmapDescriptorFactory` an argument to the controller class that you are testing, and then DI a mock version that returns mock `BitmapDescriptiors`?

Reply:
It looks like I will need to do so in two test files, as well as create and encode a minimal bitmap byte object.